br-ext/Config.in

```diff
@@ -2,6 +2,7 @@ source "$BR2_EXTERNAL_OPTEE_PATH/package/optee_os_ext/Config.in"
 source "$BR2_EXTERNAL_OPTEE_PATH/package/optee_client_ext/Config.in"
 source "$BR2_EXTERNAL_OPTEE_PATH/package/optee_test_ext/Config.in"
 source "$BR2_EXTERNAL_OPTEE_PATH/package/optee_examples_ext/Config.in"
+source "$BR2_EXTERNAL_OPTEE_PATH/package/optee_rust_ext/Config.in"
```

Comment:
Since this also is an example I believe `optee_rust_examples_ext` should be more accurate.